io.anytype.anytype.metainfo.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.anytype.anytype</id>
+
+  <name>Anytype</name>
+  <summary>The everything app for those who celebrate trust and autonomy</summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://raw.githubusercontent.com/anyproto/anytype-ts/main/LICENSE.md</project_license>
```

Comment:
Does this allow redistribution ? If not you should use extra-data source unless upstream is fine with using the original source.

Reply:
It does allow distribution in a non-commercial way:

> Any Association grants you (“Licensee”) a license to use, modify, and 
redistribute the Software, but only 
(a) for Non-Commercial Use, or (b) for Commercial Use in Allowed Networks.

And the upstream is fine with me [publishing](https://github.com/anyproto/anytype-ts/issues/74)
> we're fine with it. Let's see if the users will be satisfied with this solution or if they want it to be distributed from our account :)